packages/create-block/README.md

```diff
@@ -99,6 +99,53 @@ $ npm run packages-update
 ```
 Updates WordPress packages to the latest version. [Learn more](/packages/scripts#packages-update).
 
+## External Templates
+Since version 0.12.0 it is possible to use external templates hosted on NPM. These templates need to contain `.mustache` files that will be used in the scaffolding and one `template.json` for the metadata. 
+
+### Availabe Variables: 
+- `namespace`
+- `slug`
+- `title`
+- `textdomain`
+- `description`
+- `category`
+- `dashicon`
+- `license`
+- `licenseURI`
+- `namespaceSnakeCase`
+- `slugSnakeCase`
```

Comment:
`namespaceSnakeCase` and `slugSnakeCase` ar sort of special here. We should omit them and find a way to apply those modifications inside the template.